packages/plugins/src/SiteMapPlugin.ts

```diff
@@ -1,12 +1,13 @@
 import type { Page, Plugin as PluginType } from '@jpmorganchase/mosaic-types';
+import path from 'path';
 
 interface SiteMapPluginConfigData {
   sitemaps: Array<string[]>;
 }
 
 const SiteMapPlugin: PluginType<Page, unknown, SiteMapPluginConfigData, SiteMapPluginConfigData> = {
   // Merge together all of the individual sitemaps from each source
-  async afterUpdate(_, { sharedFilesystem, globalConfig }) {
+  async afterUpdate(_, { sharedFilesystem, globalConfig }, { siteUrl }) {
```

Comment:
Should there not be a `SiteMapPluginOptions` type somewhere with `siteUrl` specified?

Reply:
added